### PR TITLE
Removed outdated flow

### PR DESCRIPTION
**Pull request by sirLisko**
Removed flow!

Converted all `types` to `PropTypes`.

### Next step
Reenable the eslint checks about `PropTypes` and add / fix them :)

#### How to verify
1. Check out this branch
1. `npm install`
1. `cd site`
1. `npm install`
1. `npm start`
1. Check that everything looks fine in the style guides

### CODE REVIEW

File: src/components/avatar/avatar.react.js [hunk 1]
```diff
@@ -76,4 +61,27 @@ export default function Avatar(props: TProps) {
       />
     </UIBase>
   );
-}
+};
+
+Avatar.propTypes = {
+  /** A URL to the avatar */
```
[the remark this answers is not in the document]
Comment (sirLisko) [reply src/components/avatar/avatar.react.js]: The comments are generating the documentation:

Screenshot 2019-05-16 at 16 05 19

File: src/components/alert/alert.react.js [hunk 2]
```diff
@@ -61,4 +51,19 @@ export default function Alert(props: TProps) {
...
-}
+};
+
+Alert.propTypes = {
+  children: PropTypes.node,
+  /** 'info' | 'error' | 'warning' | 'confirmation' */
+  type: PropTypes.oneOf(['info', 'error', 'warning', 'confirmation'])
+    .isRequired,
+  showIcon: PropTypes.bool,
+  /** 'left' | 'center' */
```
[the remark this answers is not in the document]
Comment (sirLisko) [reply src/components/alert/alert.react.js]: So with flow `'left' | 'center'` was appearing in the `type` section of the doc. With the `PropTypes` `type` is becoming `enum`.
I thought it was meaningful keeping the option visible in the docs.

_with flow_
Screenshot 2019-05-16 at 16 08 04

_with props_
Screenshot 2019-05-16 at 16 08 13

[the remark this answers is not in the document]
Comment (sirLisko) [reply src/components/alert/alert.react.js]: Didn't check if there is a better way to achieve the same, it was already too massive this PR 😅

File: src/components/table/table-cell.react.js [hunk 3]
```diff
@@ -81,26 +80,7 @@ const StyledTableHeaderCell = styled('th')`
...
-  },
-  // Derived from props
-  header: boolean,
-  active: boolean,
-  sortable: boolean,
-  outerBorder: boolean,
-};
-
-export function TableCell(props: TTableCellProps) {
+export function TableCell(props) {
```
[the remark this answers is not in the document]
Comment (sirLisko) [reply src/components/table/table-cell.react.js]: Good catch. I will fix it :)

**Comment by mroderick** [comment]
Excellent! Looking forward to seeing this one merged!

**Comment by mroderick** [comment]
Yeah baby!

**Pull request merged**